github/github.go

```diff
@@ -961,20 +976,53 @@ func (g *GitHub) UpdateReleasePage(
 	releaseID int64,
 	tag, commitish, name, body string,
 	isDraft, isPrerelease bool,
+) (release *github.RepositoryRelease, err error) {
+	return g.UpdateReleasePageWithOptions(owner, repo, releaseID, tag, commitish, &UpdateReleasePageOptions{
+		Name:       &name,
+		Body:       &body,
+		Draft:      &isDraft,
+		Prerelease: &isPrerelease,
+	})
+}
+
+// toRepositoryRelease builds a repository release from the set of options.
+func (u *UpdateReleasePageOptions) toRepositoryRelease() *github.RepositoryRelease {
+	request := &github.RepositoryRelease{}
+	request.Name = u.Name
+	request.Body = u.Body
+	request.Draft = u.Draft
+	request.Prerelease = u.Prerelease
+
+	if u.Latest != nil {
+		if *u.Latest {
+			request.MakeLatest = ptr.To("true")
+		} else {
+			request.MakeLatest = ptr.To("false")
+		}
+	}
```

Comment:
Why do we need this? Why can't we apply the same logic as for `Draft` and `Prerelease`?

Reply:
`MakeLatest` is a string, so I needed to translate a boolean into a string here. I didn't want to expose a string because that's easy to get wrong, and the third option is `"legacy"`, wich did not look like something we need.